bitnami/redis/templates/configmap.yaml

```diff
@@ -83,12 +83,15 @@ data:
       sentinel client-reconfig-script {{ .Values.sentinel.masterSet }} /opt/bitnami/scripts/start-scripts/push-master-label.sh
     {{- end }}
     {{- if .Values.auth.acl.sentinel }}
-    {{- range .Values.auth.acl.users }}
+    {{- $userSecret := .Values.auth.acl.userSecret -}}
```

Comment:
oh no! blunder! this templates the following line's comment _right_ after the previous line, which is incorrect:

```
sentinel parallel-syncs redis-prod 1# Sentinel ACL configuration, only for users with password
```

removing the `-` ad the end here should be sufficient to fix this. I'll PR a fix